assets/js/feature-tours/ga4-reporting.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/*
+ * Internal dependencies
+ */
+import { VIEW_CONTEXT_MAIN_DASHBOARD } from '../googlesitekit/constants';
+import Link from '../components/Link';
+
+const ga4Reporting = {
+	slug: 'ga4Reporting',
+	contexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+	version: '1.98.0',
```

Comment:
This is set in the IB but we probably want to use a farther future version to ensure people see it when setting up. This is also relevant for tours that are for features that are behind a flag, meaning we don't actually know which version of Site Kit the feature will be effectively "released" in, as this can be different depending on when the feature is granted. So we should probably use a far future version here until GA4 reporting is rolled out to 100%.

Reply:
Updated to be `1.999.0` for now and left a comment to change it to the actual version when we are ready.

Reply:
Actually, I think the version doesn't really matter if we run the feature tour on demand... 🤔

Reply:
> I think the version doesn't really matter if we run the feature tour on demand... 

You're right, it doesn't – I left this comment before I realized it was only being triggered on demand.